lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -162,14 +162,10 @@ def __init__(self, *args):
 the continuity of the metrics time series gathered by Prometheus when the leader unit
 changes (e.g. on upgrade or rescale).
 
-It is also possible to configure other scrape related parameters using
-these job specifications as described by the Prometheus
-[documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
-The permissible subset of job specific scrape configuration parameters
-supported in a `MetricsEndpointProvider` job specification are:
+Additionally, it is also technically possible, but **strongly discouraged**, to
+configure the following scrape-related settings, which behave as described by the
+[Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config):
```

Comment:
This is possibly a reasonable case for shortcut reference links in Markdown.

Alternatively, I suspect that the following will actually look "normal":
```python
configure the following scrape-related settings, which behave as described by the
 [Prometheus documentation](
 https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config):
```

Same for below.

Reply:
Do I feel like I should take the chance to see how borked markdown references are in the CharmHub / Discourse rendering? Make a guess :-)